test/helpers/data/workload-capacity-helper.js

```diff
@@ -2,109 +2,170 @@ const config = require('../../../knexfile').integrationTests
 const knex = require('knex')(config)
 var Promise = require('bluebird').Promise
 
+var defaultWorkload = {
+  total_cases: 5,
+  total_community_cases: 0,
+  total_custody_cases: 0,
+  total_license_cases: 0,
+  monthly_sdrs: 0,
+  sdr_due_next_30_days: 0,
+  sdr_conversions_last_30_days: 0,
+  paroms_completed_last_30_days: 0,
+  paroms_due_next_30_days: 0,
+  license_last_16_weeks: 9,
+  community_last_16_weeks: 10
+}
+
+var defaultWorkloadPoints = {
+  comm_tier_1: 0,
+  comm_tier_2: 0,
+  comm_tier_3: 0,
+  comm_tier_4: 0,
+  comm_tier_5: 0,
+  comm_tier_6: 0,
+  comm_tier_7: 0,
+  cust_tier_1: 0,
+  cust_tier_2: 0,
+  cust_tier_3: 0,
+  cust_tier_4: 0,
+  cust_tier_5: 0,
+  cust_tier_6: 0,
+  cust_tier_7: 0,
+  lic_tier_1: 0,
+  lic_tier_2: 0,
+  lic_tier_3: 0,
+  lic_tier_4: 0,
+  lic_tier_5: 0,
+  lic_tier_6: 0,
+  lic_tier_7: 0,
+  user_id: 0,
+  sdr: 0,
+  sdr_conversion: 0,
+  nominal_target_spo: 0,
+  nominal_target_po: 0,
+  default_contracted_hours_po: 0,
+  default_contracted_hours_pso: 0,
+  weighting_o: 0,
+  weighting_w: 0,
+  weighting_u: 0,
+  paroms_enabled: 0,
+  parom: 0
+}
+
+module.exports.addCaseProgressDataForAllOrgUnits = function () {
+  return module.exports.addWorkloadCapacitiesForOffenderManager()
+  .then(function (inserts) {
+    return addOffenderManager(inserts)
+  })
+  .then(function (inserts) {
+    return addTeam(inserts)
+  })
+  .then(function (inserts) {
+    return addOffenderManager(inserts)
+  })
+  .then(function (inserts) {
+    return addLdu(inserts)
+  })
+  .then(function (inserts) {
+    return addTeam(inserts)
+  })
+  .then(function (inserts) {
+    return addOffenderManager(inserts)
+  })
+  .then(function (inserts) {
+    return addPopulatedRegion(inserts)
```

Comment:
Can we apply this pattern of adding the children when we add a parent organisation?